app/dashboards/redirect_dashboard.rb

```diff
@@ -9,7 +9,7 @@ class RedirectDashboard < Administrate::BaseDashboard
   # on pages throughout the dashboard
 
   def self.transform_to_option_data(collection, type)
-    collection.collect { |r| [r.title, "#{(type == "lesson") ? r.unit.slug : r.year_group.key_stage.slug},#{r.slug}"] }
+    collection.collect { |r| ["#{r.title} - #{r.slug}", "#{(type == "lesson") ? r.unit.slug : r.year_group.key_stage.slug},#{r.slug}"] }
```

Comment:
Just showing the slug on the drop down was a bit hard to read with a long list of redirects. I had a chat with Vicky and we both agreed showing the title and slug was a bit more readable.